### ISSUE TITLE
unable to install docker on windows 10 pro version 1511

### ISSUE THREAD

**Issue by daemonslayer** [docker/for-win#1421]
### Expected behavior
According to the documentation, docker should have installed. I have correctly setup : 
1. Hyper-V enabled
2. Windows Updates (Version 1511 (Build 10586.1176))

![index](https://user-images.githubusercontent.com/10099126/33924085-7394adc2-dffa-11e7-8d98-84296cf6ccc6.jpg)

### Actual behavior
Error.
![error](https://user-images.githubusercontent.com/10099126/33924076-614efbf4-dffa-11e7-8ad1-93a55cb38b3e.jpg)

### Information

I have been able to setup docker using virtualbox by disabling hyper-v. But I want to run docker for windows. I dont know why its not being able to run.

**Comment by friism** [comment]
Windows 10 build 10586 is old, and we recommend upgrading.

If you cannot upgrade, you can install older Docker for Windows versions. 17.06 should install: https://docs.docker.com/docker-for-windows/release-notes/#docker-community-edition-17062-ce-win27-2017-09-06-stable

**Comment by MarkStoakes** [comment]
I encountered the same issue.  "Windows 10 build 10586 is old, and we recommend upgrading." seems an odd response when the docker prerequisites require this version.

**Comment by gideondsouza** [comment]
I don't get it ? I have windows 10 Home Single language Os build 16299.192. I updated yesterday.

And docker says : 
`Docker for Windows requires Windows 10 Pro or Enterprise version 10586, or Windows server 2016 RTM to run`


So what is the deal.

Is docker for some reason not supported on my machine?

**Comment by mcse3010** [comment]
@gideondsouza Docker for Windows is only supported on Windows 10 Pro or Enterprise, Windows 10 Home is not supported.  You can upgrade your Home to Pro in your System control panel.

**Comment by ph4wks** [comment]
I have Windows 10 Enterprise, Hyper-V enabled and am on 10586. Old or not this is the minimum requirement and so should work. It doesn't and fails with the error mentioned by everyone above.

**Comment by dheemanth1434** [comment]
I have installed windows 10 education on my pc when I tried to start docker
![2018-03-18](https://user-images.githubusercontent.com/22279025/37568446-314fbd3e-2afb-11e8-84e7-75449b00f383.png)
 it is showing windows version is not supported, please resolve.

**Comment by rn** [comment]
[the documentation](https://docs.docker.com/docker-for-windows/install/#download-docker-for-windows) states:
```
The current version of Docker for Windows runs on 64bit Windows 10 Pro, Enterprise and Education (1607 Anniversary Update, Build 14393 or later)
```

**Comment by rafael-yure** [comment]
This is also happening to me:

![docker](https://user-images.githubusercontent.com/8876325/37575914-5fa59f84-2b08-11e8-9630-8370e532923f.png)

I have Win 10 Pro, Hyper-V enabled and the correct build, but it won' install.

**Comment by dheemanth1434** [comment]
my windows version is 1607 but it is not working
![build](https://user-images.githubusercontent.com/22279025/37578803-0773d480-2b61-11e8-8305-b2323b1d7202.JPG)

**Comment by rn** [comment]
@rafael-yure the error message was wrong and should be fixed in the latest 17.12 update.

@dheemanth1434 please provide details on what is not working ideally on a new issue instead of an issue which has been closed. Thanks
